code/game/objects/items/weapons/blades.dm

```diff
@@ -136,22 +136,29 @@
 
 	switch(loaded_reagent.type)
 		if(/datum/reagent/medicine/tramadol)
-			M.apply_status_effect(/datum/status_effect/incapacitating/harvester_slowdown, 2 SECONDS)
+			M.apply_damage(force*0.6, BRUTE, user.zone_selected)
+			M.apply_status_effect(/datum/status_effect/incapacitating/harvester_slowdown, 1 SECONDS)
 
 		if(/datum/reagent/medicine/kelotane)
-			var/turf/target = get_turf(M)
-			target.ignite()
 			var/list/cone_turfs = generate_cone(user, 2, 1, 91, Get_Angle(user, M.loc))
 			for(var/X in cone_turfs)
 				var/turf/T = X
-				T.ignite()
+				for(var/mob/living/C in T)
+					C.flamer_fire_act(10)
+					C.apply_damage(20, BURN, user.zone_selected, C.get_soft_armor("fire", user.zone_selected))
+					if(C.IgniteMob())
+						C.visible_message("<span class='danger'>[C] bursts into flames!</span>","[isxeno(C)?"<span class='xenodanger'>":"<span class='highdanger'>"]You burst into flames!</span>")
 
 		if(/datum/reagent/medicine/bicaridine)
-			to_chat(user, "<span class='rose'>You prepare to stab [M]!</span>")
-			if(!do_after(user, 2 SECONDS, TRUE, M, BUSY_ICON_DANGER))
-				return FALSE
+			if(isxeno(M))
+				return ..()
+			to_chat(user, "<span class='rose'>You prepare to stab <b>[M != user ? "[M]" : "yourself"]</b>!</span>")
 			new /obj/effect/temp_visual/telekinesis(get_turf(M))
-			M.heal_overall_damage(10, 0, TRUE)
+			if((M != user) && do_after(user, 2 SECONDS, TRUE, M, BUSY_ICON_DANGER))
+				M.heal_overall_damage(12.5, 0, TRUE)
+			else
+				M.adjustStaminaLoss(-30)
+				M.heal_overall_damage(6, 0, TRUE)
```

Comment:
2 sec for 12.5? Come on slot, why would you use this rather than just giving the bicaridine directly

Reply:
It costs 5u, is usable on dead people for revives and heals about 6.25 per tick versus bicaridine's 1/1.5 per tick. I could make it better later if need be, but I feel like if it is better it effectively makes marines heal much faster than xenos.